_posts/2015-05-16-prism-break.md

```diff
@@ -0,0 +1,31 @@
+
+Alas, the cuttlefish has but one type of photoreceptor. The instantaneous color and texture changes exhibited by this mollusk allow it to blend in seamlessly with any background it encounters, but it is effectively color blind, at least if color vision depends on comparison between cell types. [Some new research](http://dx.doi.org/10.1101/017756) suggests that "color blind" cephalopods like the cuttlefish _can_ sense chromatic information using an optical trick.
+
+![Eye of Australian giant cuttlefish (Sepia apama)]({{ site.url }}/images/2015-05-16-prism-break.jpg)
+
+> _Eye of Australian giant cuttlefish (Sepia apama). [Photo](https://www.flickr.com/photos/rling/5101121081/in/photostream/) by Richard Ling / [CC BY-NC-ND 2.0](https://creativecommons.org/licenses/by-nc-nd/2.0/)_
+
+Cuttlefish eyes make use of some interesting properties of light. Specifically, their pupils are unusual W-shaped slits that amplify chromatic aberration, the way light of different wavelengths bends slightly differently. Chromatic aberration explains purplish fringes sometimes seen in photographs: short wavelengths (blue light) get refracted or bent more than long wavelengths (red light)[^1]. Eliminating these "errors" is hard in any optical system, whether it's eyes or cameras. For the cuttlefish, this aberration signal actually provides valuable color information that is undetectable with only a single photoreceptor type.
+
+Unlike the round human pupil, the squiggly pupil of the cuttlefish lets in light peripherally, off of the optical axis. This means the chromatic effects -- how differently the incoming wavelengths are focused -- are enhanced. Through its normal accommodation (focusing), the cuttlefish can _infer_ the colors of the environment, effectively using the distance it needs to focus as a clue to color.
+
+For this system to work, the environment of the cuttlefish has to be full of cues to help it accommodate, like shadows and texture. Moreover, these features have be fairly sharply segregated spectrally (that is, neighboring objects need to be different and non-overlapping in hue), and fortunately for the cuttlefish they are. With its ersatz color vision mechanism the cuttlefish can't distinguish a broad flat field of a single color, though.
```

Comment:
color-vision? I really think it should be hyphenated here and above, but the decision is yours.